pkg/controllermanager/controller/shoot/shoot_control_reconcile.go

```diff
@@ -236,6 +236,11 @@ func (c *defaultControl) reconcileShoot(o *operation.Operation, operationType ga
 			Fn:           flow.SimpleTaskFn(botanist.DeployCertBroker).RetryUntilTimeout(defaultInterval, defaultTimeout),
 			Dependencies: flow.NewTaskIDs(initializeShootClients, deployKubeAddonManager),
 		})
+		_ = g.Add(flow.Task{
+			Name:         "Scaling control plane",
```

Comment:
Hmm this is actually hibernating the control plane... I know you do this to not confuse a user but still I'd rather have a name like `Hibernating Control plane` or a name that depends on the `o.Shoot.IsHibernated` attribute - otherwise users might think that this really scales the control plane e.g. depending on requests.